addons/sale_management/models/sale_order_option.py

```diff
@@ -38,9 +38,7 @@ class SaleOrderOption(models.Model):
         string="Unit of Measure",
         compute='_compute_uom_id',
         store=True, readonly=False,
-        required=True, precompute=True,
-        domain="[('category_id', '=', product_uom_category_id)]")
-    product_uom_category_id = fields.Many2one(related='product_id.uom_id.category_id')
+        required=True, precompute=True)
```

Comment:
I would assume that it would require the same kind of domain to check the `allowed_uom_ids`, but quite frankly I have no idea how/when this field is used, so may not be necessary? :thinking: